src/mvn.ts

```diff
@@ -0,0 +1,259 @@
+const completionSpec: Fig.Spec = {
+  name: "mvn",
+  description: "Maven - a Java based project management and comprehension tool",
+  options: [
+    {
+      name: ["--also-make", "-am"],
+      description: "Also build projects required by project list",
+    },
+    {
+      name: ["--also-make-dependents", "-amd"],
+      description:
+        "Also build projects that depend on projects in the project list",
+    },
+    {
+      name: ["--batch-mode", "-B"],
+      description: "Run in non-interactive (batch)",
+    },
+    {
+      name: ["--builder", "-b"],
+      description: "Specify the build strategy to use",
+      args: {
+        name: "id of build strategy",
+      },
+    },
+    {
+      name: ["--strict-checksums", "-C"],
+      description: "Fail if checksums do not match",
+    },
+    {
+      name: ["--lax-checksums", "-c"],
+      description: "Warn if checksums do not match",
+    },
+    {
+      name: "--color",
+      description: "Specify the color mode of the output",
+      args: {
+        name: "color mode",
+        suggestions: ["always", "never", "auto"],
+      },
+    },
+    {
+      name: ["--check-plugin-updates", "-cpu"],
+      description: "Ineffective. Only kept for backward compatibility",
+    },
+    {
+      name: ["--define", "-D"],
+      description: "Define a system property",
+      args: {
+        name: "system property",
+      },
+    },
+    {
+      name: ["--errors", "-e"],
+      description: "Produce execution error messages",
+    },
+    {
+      name: ["--encrypt-master-password", "-emp"],
+      description: "Encrypt the master security password",
+      args: {
+        name: "master password",
+      },
+    },
+    {
+      name: ["--encrypt-password", "-ep"],
+      description: "Encrypt the server password",
+      args: {
+        name: "server password",
+      },
+    },
+    {
+      name: ["--file", "-f"],
+      description:
+        "Force the use of an alternate POM file (or directory with pom.xml)",
+      args: {
+        name: "path",
+        suggestions: ["filepaths", "folders"],
```

Comment:
This should actually be:

```suggestion
        template: ["filepaths", "folders"],
```

Reply:
I blame GitHub Copilot 😂 

Fixed.